raspiraw.c

```diff
@@ -818,7 +821,9 @@ int main(int argc, const char** argv) {
 
 	if (argc == 1)
 	{
-		fprintf(stdout, "\n%s Camera App %s\n\n", basename(argv[0]), VERSION_STRING);
+		char *buf = malloc(strlen(argv[0])+1);
+		strcpy(buf, argv[0]);
+		fprintf(stdout, "\n%s Camera App %s\n\n", basename(buf), VERSION_STRING);
```

Comment:
Sorry, wrong approach on both. You've got a memory leak of ```buf``` here too.
I've pushed fixes to them to my branch.